command/meta.go

```diff
@@ -0,0 +1,202 @@
+package command
+
+import (
+	"bufio"
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/mitchellh/cli"
+	text "github.com/tonnerre/golang-text"
+)
+
+// maxLineLength is the maximum width of any line.
+const maxLineLength int = 72
+
+// FlagSetFlags is an enum to define what flags are present in the
+// default FlagSet returned.
+type FlagSetFlags uint
+
+const (
+	FlagSetNone FlagSetFlags = iota << 1
+	FlagSetHTTP FlagSetFlags = iota << 1
+)
+
+type Meta struct {
```

Comment:
should we rename it to `CommandMeta` or something similar? I agree the name's not great